english/marshmallow.md

```diff
@@ -232,7 +232,7 @@ You pause for a second, awaiting a spectacular sight.
 
 Sheepishly you look down on the crappy wand in your hand.
 
-You must be doing something wrong. You drink a beer and think about it, then Ah-ha!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.